tests/Temporalio.Tests/WorkflowEnvironmentTestBase.cs

```diff
@@ -20,4 +42,33 @@ protected WorkflowEnvironmentTestBase(ITestOutputHelper output, WorkflowEnvironm
     protected WorkflowEnvironment Env { get; private init; }
 
     protected ITemporalClient Client { get; private init; }
+
+    protected async Task EnsureSearchAttributesPresentAsync()
+    {
+        // Only add search attributes if not present
+        var resp = await Client.Connection.WorkflowService.GetSearchAttributesAsync(new());
+        if (resp.Keys.ContainsKey(AttrBool.Name))
+        {
+            return;
+        }
+        await Client.Connection.OperatorService.AddSearchAttributesAsync(new()
```

Comment:
Obviously not for this PR but we could have a high level API for this too now that we have typed attributes.

Reply:
That can be another proposal or approved change. Registering search attributes it's not part of this effort by intention here or other SDKs when I implement there.